posts/2023-12-21_end_of__year__up.../end_of__year__update_from_the__pharmaverse__council.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "End of Year Update from the Pharmaverse Council"
+author:
+  - name: Mike Stackhouse
+description: ""
+date: "2023-12-21"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-12-21_end_of__year__up..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+## 2023 Was a Big Year
+
+This was a big year for open-source work in clinical submissions in general. We saw Roche speak about shifting to an [open-source backbone for clinical trials](https://www.youtube.com/watch?v=nqJsLSLd39A). Novo Nordisk spoke publicly of an [R based submission to the FDA](https://www.youtube.com/watch?v=t33dS17QHuA). These are true marks of progress being made in R becoming a first class language for clinical reporting. 
+
+Back in August, Nicholas Eugenio released a [blog post](https://pharmaverse.github.io/blog/posts/2023-10-10_pharmaverse_story/pharmaverse_story.html) on this history of pharmaverse. It's funny to think about the fact that only 3 years ago, the idea of cross organization collaboration on R packages and building a community around this in the clinical world was just a conversation between friends. Since then, we have a community of more than 1200 people on Slack, 350 on LinkedIn, interest from over 150 organizations, and over 30 packages. If 2020 through 2022 was the birth of Pharmaverse, 2023 was finding our identity as a community. For the council, 2024 will be about continuing to mature and find more ways that we can continue to support the community.
```

Comment:
```suggestion
Back in August, Nicholas Eugenio released a [blog post](https://pharmaverse.github.io/blog/posts/2023-10-10_pharmaverse_story/pharmaverse_story.html) on the history of pharmaverse. It's funny to think about the fact that only 3 years ago, the idea of cross organization collaboration on R packages and building a community around this in the clinical world was just a conversation between friends. Since then, we have a community of more than 1200 people on Slack, 350 on LinkedIn, interest from over 150 organizations, and over 30 packages. If 2020 through 2022 was the birth of Pharmaverse, 2023 was finding our identity as a community. For the council, 2024 will be about continuing to mature and find more ways that we can continue to support the community.
```